src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Base64Url/Base64UrlDecoder.cs

```diff
@@ -675,7 +675,11 @@ public unsafe bool TryLoadVector512(ushort* src, ushort* srcStart, int sourceLen
                     return false;
                 }
 
+#if NET9_0_OR_GREATER
```

Comment:
Why we need this? The main branch is already .net 10 right?

Reply:
This file is partially shared between CoreLib and the NuGet package (e.g. see a couple lines above this change). Otherwise you're right and we wouldn't need it here.